json_schema/module/biomaterial/reproduction_history.json

```
@@ -0,0 +1,57 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "description": "Reproduction history of the donor.",
```

Would 

```suggestion
    "description": "Reproductive history of the donor.",
```

be better? And similarly for the file name and the property referencing it.